client/scss/components/_list_card.scss

```diff
@@ -0,0 +1,105 @@
+//* @define list-card
+
+$list-card-number-size: 96px;
+
+.list-card {
```

Comment:
What do we mean by `card`?
Is that what we're calling things that are in themes? We have something called a `promo` too?